packages/react/src/components/uiKit/blocks/UiKitModal.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable react/jsx-no-undef */
```

Comment:
Instead of disabling the errors. It will be better to comment out where the imported functions or components are being used and add label "to do" as comment.

Reply:
Oops ! It's been merged now, what to do ? @abhinavkrin

Reply:
No worries Raise another PR for this.

Reply:
Okay Sure